app/Helpers/database/subscription.php

```diff
@@ -160,7 +180,7 @@ function getSubscribersOfArticle(
         SELECT _ua.*
         FROM UserAccounts as _ua
         WHERE (_ua.User = :subjectAuthor OR _ua.display_name = :subjectAuthor)
-              $websitePrefsFilter
+              AND (_ua.websitePrefs & :websitePrefs) != 0
```

Comment:
This change seems possibly wrong. Previously, if `$noExplicitSubscriptions` was false, this subclause would be blank.

If it is desirable to stil have it, then `$websitePrefsFilter` should be eliminated.

Reply:
Confirmed. Latest sets this line back to using `$websitePrefsFilter`.